### ISSUE TITLE
Dev: Send email that announces when organization information has changed

### ISSUE THREAD

**Issue by Katherine-MN** [cisagov/manage.get.gov#2832]
### Issue description

We need to send org admins an announcement when edits have been made to organization information. This includes updates to:

- org name and mailing address
- senior official

### Acceptance criteria

- [x] When a user updates info on the following pages via the registrar, an email is sent to all admins for that portfolio.
  - [x] Organization name and mailing address 
  - [x] Senior official (although this is not editable by initial users of org model [federal orgs], it may be editable by future users who are non-federal; therefore, we should account for that)
- [x] The email should use content outlined in [this document](https://docs.google.com/document/d/1GOzC-H72QCCR6Dx8bNlf9is7qxF_xibdyd_oGDBhKso/edit?tab=t.f4fjg84oz1).
- [x] Each admin should receive a separate email where the “to” line and greeting is directly addressed to them.
- [x] Do not send email if the updates were initiated in the Django admin.

### Additional context

NOTE: In org model, this information is managed at the portfolio level. That's why we're sending a unique email in this use case. For non-org model, updates to this information are announced at the domain level and use a separate email (addressed in #2307 )

**Comment by h-m-f-t** [comment]
For the avoidance of doubt: the behavior we'd want is for the person who initiated the change to be on `To:` and everyone else to be on `CC:`. If for some reason that's complicated, we'd want to avoid having _no one_ on `To:` (but putting everyone on `To:` is also workable.

Clear as mud?

**Comment by abroddrick** [comment]
@h-m-f-t and @Katherine-Osos I think we recently (monday) changed the emailing so that admins would each get their own email and not be on the CC list, is that what we want to do here as well? Adding refinement just so I don't forget this one may need a slight tweak.

**Comment by Katherine-Osos** [comment]
@abroddrick Yes! We're following that pattern here, as well. I already updated the AC and Google Doc to reflect that change - did I miss something that's causing some confusion, though?
